lib/diagnostic.js

```diff
@@ -5,8 +5,16 @@
 // the minimum value in an array
 // this callback will be passed by the tests to the reduce method, e.g.
 // `let minimum = array.reduce(min);`
-const min = function (previous, current) {
 
+const array = [1, 2, 4, 3]
+
+const min = function (previous, current) {
+  let minimum = reduce(array, (previous, current) => {
```

Comment:
Cool 👍 . Although you don't have to have a `reduce` inside your function, since the function that you write will be passed to another `reduce` function per the prompt.